src/main/resources/templates/client/cypher/list.html

```diff
@@ -57,5 +54,23 @@ <h1>Šifry</h1>
                 </a>
             </div>
         </div>
+    </div>
+    <a th:href="@{/finalplace}" id="footer">
+        <div id="final-place">
+            <div class="container">
+                <div class="row align-items-center">
+                    <div id="time" class="col">
+                        18:00
```

Comment:
Tohle stačí natvrdo momentálně?

Reply:
Ajo, zapomnel jsem napsat. Udelal jsem jen UI a pockam, az se mergne ten pavluv detail toho final place, aby se dala vyuzit logika, co tam bude mit. (Ted je to totiz zobrazene porad)